feature-libs/asm/root/facade/asm-bind-cart.facade.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { facadeFactory } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { ASM_FEATURE } from '../feature-name';
+
+@Injectable({
+  providedIn: 'root',
+  useFactory: () =>
+    facadeFactory({
+      facade: AsmBindCartFacade,
+      feature: ASM_FEATURE,
+      methods: ['bindCart'],
+    }),
+})
+export abstract class AsmBindCartFacade {
+  /**
+   * Bind an anonymous cart to a registered user
+   * @param userId The user to associate a cart with.  Defaults to the current user if not provided.
```

Comment:
The doc comment needs updating to remove the userId param (and add cartid perhaps)